runscanner/src/main/java/uk/ac/bbsrc/tgac/miso/runscanner/processors/DefaultIllumina.java

```diff
@@ -36,10 +36,24 @@
 
   private static final Logger log = LoggerFactory.getLogger(DefaultIllumina.class);
 
+  private static final XPathExpression RUN_COMPLETION_STATUS_EXPRESSION = compileXPath("//CompletionStatus")[0];
+
   public static DefaultIllumina create(Builder builder, ObjectNode parameters) {
     return new DefaultIllumina(builder);
   }
 
+  private static Optional<HealthType> getHealth(Document document) {
+    try {
+      switch ((String) RUN_COMPLETION_STATUS_EXPRESSION.evaluate(document, XPathConstants.STRING)) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a default case to this switch. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ASwitchLastCaseIsDefaultCheck)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this "switch" statement by "if" statements to increase readability. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1301)